src/components/Staking/StakingStatsBox.js

```diff
@@ -96,7 +96,7 @@ const StatsBoxGrid = () => {
         const valueCurrentApr = formatPercentage(currentAprDecimal)
         setTotalEth(valueTotalEth)
         setTotalValidators(valueTotalValidators)
-        setCurrentApr(valueCurrentApr)
+        setCurrentApr(`~${valueCurrentApr}`)
```

Comment:
Not a huge deal until we migrate to TypeScript, but we're doing an implicit type conversion from int to string here

Reply:
The `formatPercentage` helper function returns a string already before this is passed to the state setter. This doesn't actually add any implicit conversion, but yeah... TS will soon help make all of this more clear.